bin/setup_spark.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Latest stable release 
+SPARK_VERSION="3.5.0"
```

Comment:
Quotes are unnecessary unless you have variables (e.g. `$HOME`) on the right hand side.